src/components/shared/CustomImage.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box } from '@mui/material';
+import { useEffect, useState } from 'react';
+import LoadingImage from '../../assets/loader.png';
+import ErrorImage from '../../assets/no-image.png';
+
+const imageStates = {
+  Loading: LoadingImage,
+  Error: ErrorImage,
+  Loaded: '',
+};
+
+type ImageState = keyof typeof imageStates;
+type SingleImageProps = { image: string; name?: string };
+
+function CustomImage({ image, name = 'image' }: SingleImageProps) {
```

Comment:
JFYI, it is not the best solution to have that simple `image` value as alt attribute. 

in short, `alt` has to have meaningful text that describes image or be skipped if image is decorative 

https://blog.hubspot.com/marketing/image-alt-text

Reply:
There is no need to fix it now because our app doesn't have any a11y constraints